src/ophys_etl/transforms/suite2p_wrapper.py

```diff
@@ -0,0 +1,219 @@
+import argschema
+import suite2p
+import pathlib
+import marshmallow as mm
+import h5py
+import tempfile
+from typing import List
+import datetime
+import shutil
+
+
+class Suite2PWrapperException(Exception):
+    pass
+
+
+class Suite2PWrapperSchema(argschema.ArgSchema):
+    # Options for running s2p. One list can be found at:
+    # https://github.com/MouseLand/suite2p/blob/master/suite2p/run_s2p.py
+
+    # s2p IO settings (file paths)
+    h5py = argschema.fields.InputFile(
```

Comment:
I think a better name would be 'motion_corrected_video_path', h5py doesn't describe the input but the format.

Reply:
we don't want to rename Suite2P variables.